src/index.ts

```diff
@@ -56,6 +46,35 @@ const plugin: PluginImpl<{}> = () => {
     return { source, program };
   }
 
+  // Parse a TypeScript module into an ESTree program.
+  const references = new Set<string>();
```

Comment:
I don’t really have a better idea right now, just keep in mind that introducing more *global state* makes it harder to fix watch mode (#35)